logicrunner/executionbroker_test.go

```diff
@@ -673,8 +673,6 @@ func TestExecutionBroker_OnPulse(t *testing.T) {
 				objectRef := gen.Reference()
 				broker := NewExecutionBroker(objectRef, nil, nil, nil, nil, nil, nil, nil, nil)
 				broker.pending = insolar.InPending
-				broker.requestsFetcher = NewRequestsFetcherMock(t).
-					FetchPendingsMock.Return()
```

Comment:
I can't believe this is correct, especially considering comment to this test. this test was assuring that we start requestsFetcher and you delete exactly this check.